estate/models/buildings_model.py

```diff
@@ -0,0 +1,83 @@
+from odoo import models, fields, api
+from odoo.exceptions import UserError
+from datetime import timedelta
+
+
+class Building(models.Model):
+    _name = "estate.buildings"
+    _description = "Buildings Model"
+
+    name = fields.Char()
+    description = fields.Text()
+    value = fields.Integer(copy=False)
+    availability_date = fields.Date(
+        default=lambda self: fields.Date.today() + timedelta(days=90), copy=False
+    )
+    number_of_rooms = fields.Integer(default=2)
+    garden_area = fields.Integer()
+    building_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        [("north", "North"), ("south", "South"), ("east", "East"), ("west", "West")],
+        "garden Orientation",
+    )
+    active = fields.Boolean(default=True)
+    state = fields.Selection(
+        [
+            ("new", "New"),
+            ("offer received", "Offer Received"),
```

Comment:
Use snake_case for variables and technical keys
```suggestion
            ('offer_received', "Offer Received"),
```